### ISSUE TITLE
Add riak init scripts to `chkconfig` on RedHat based distros

### ISSUE THREAD

**Issue by lukebakken** [basho/node_package#110]
The following was used on RHEL 6 to include the `/etc/init.d/riak` and other init scripts to the init management system. This makes it easier to manage the various symlinks in the `rc.X` directories to enable/disable these services.

```
# chkconfig --add riak
# chkconfig --add riak-cs
# chkconfig --add stanchion
```

**Comment by jaredmorrow** [comment]
So do you recommend adding something to the top of the init script to do this for the user?

**Comment by jaredmorrow** [comment]
Or maybe the post area of the spec file?

**Comment by lukebakken** [comment]
I think part of the install process of the rpm... so I'm assuming that's what the "post area of the spec file" means :smile:

**Comment by jaredmorrow** [comment]
Agreed.  I'll get that added before the beta tag.  Thanks.

**Comment by jaredmorrow** [comment]
This was fixed in #112
